### ISSUE TITLE
Blinkmove

### ISSUE THREAD

**Issue by ThaUnknown** [ImpactDevelopment/ImpactClient#868]
General idea is to have a module that works like blink but automatically at an interval to emulate the player lagging and maby have it interrupted on any form of interaction like killaura attacking or opening chests

**Comment by VitorISs** [comment]
I THINK there's already a module called fakelag that kinda does it.. i'm not sure tho
and also, couldn't you just keep turning on and turning off the blink module?

**Comment by ThatGrzegorzB** [comment]
no, it delays them, not makes them "pulse", this would delay packets, just block them for X amt of time then send them all at once

**Comment by SylveonBottle** [comment]
Fake Lag does exactly this. It holds onto the packets for up to 2 seconds. Bypasses pretty nicely, and you can disable it to appear to suddenly teleport forward and around corners.

Just make sure Fake Lag is set to "real" or whatever the option is called.

**Comment by ThatGrzegorzB** [comment]
> Fake Lag does exactly this. It holds onto the packets for up to 2 seconds. Bypasses pretty nicely, and you can disable it to appear to suddenly teleport forward and around corners.
> 
> Just make sure Fake Lag is set to "real" or whatever the option is called.

NO. It DELAYS packets and not make them behave how I described above

**Comment by SylveonBottle** [comment]
My bad, when you said "it delays them, not makes them 'pulse'", I thought you were talking about your suggestion.

Doesn't Fake Lag also do this, when set to the second mode..?

**Comment by ThatGrzegorzB** [comment]
> My bad, when you said "it delays them, not makes them 'pulse'", I thought you were talking about your suggestion.
> 
> Doesn't Fake Lag also do this, when set to the second mode..?

NOOOO IT DELAYS THE PACKETS
